### PR TITLE
feat: rpc node management for optimism

### PR DESCRIPTION
**Pull request by lukicenturi**
Part of #5414
## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: frontend/app/src/components/settings/general/rpc/EvmRpcNodeManager.vue [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+                <v-tooltip v-if="!item.owned" top open-delay="400">
+                  <template #activator="{ on, attrs }">
+                    <v-icon v-bind="attrs" v-on="on"> mdi-earth </v-icon>
+                  </template>
+                  <span>{{ tc('evm_rpc_node_manager.public_node') }}</span>
+                </v-tooltip>
+                <v-tooltip v-else>
+                  <template #activator="{ on, attrs }">
+                    <v-icon v-bind="attrs" v-on="on"
+                      >mdi-account-network</v-icon
```
Comment (kelsos) [inline frontend/app/src/components/settings/general/rpc/EvmRpcNodeManager.vue:191]: nit, can we make sure that we don't have a > in the new line?

**Review by kelsos — COMMENTED** [review]
Thanks @lukicenturi I like the new component approach.

I left a couple of comments and also at least for me it didn' properly work for ETH 
image

**Comment by lukicenturi** [comment]
> Thanks @lukicenturi I like the new component approach.
> 
> I left a couple of comments and also at least for me it didn' properly work for ETH image

Is there any error in the console, it seems to works fine here

**Comment by kelsos** [comment]
Let me check again.

**Review by kelsos — APPROVED** [review]
Thanks, @lukicenturi, I did a fresh checkout and it now seems to work fine. 

I am gonna merge but maybe in another PR can we do something about the vertical jumping of the UI when changing tabs.

**Pull request merged**